contents/founders/write-great-job-ads.md

```diff
@@ -0,0 +1,144 @@
+
+Instead, do this:
+
+- Hiring manager write the _entire_ job spec. They can copy previous ads. It is ok if they are not a great writer - they just need to write down as much as possible.
+- Get a good writer to edit the ad and make it a _good piece of written content._ If your hiring manager is already a good writer, you can skip this step.
+- Recruiter does a hygiene check for dumb stuff or anything obviously missing.
+
+**Be genuine, or even weird**
+
+Job applicants are sometimes reading dozens of ads in the course of their search, so cutting through the noise is really important - I'm assuming you don't already have some incredibly well-known brand like Google etc. 
+
+Ideally, write in the tone of voice of your company. Tell a story. Your ads don't have to follow the exact same structure every time - engineers care about different things to designers. Sometimes you will want to get super specific about exactly what you want the person to do. Other times you may have a vague sense, but you need help figuring out the role - both of these are ok, but the ads should be written differently. 
+
+I think it's actually ok to write quite a lot of content here. 'Best practice' is to make job specs as short as possible because the internet has wrecked our attention spans, but I think that just optimizes for total application volume over quality. The right candidate will spend lots of time reading and re-reading your spec because it's full of all the things they care about, and in turn you'll probably get fewer, but higher quality, applications. 
+
+**'Who this role is _not_ for'**
+
+This has been incredibly helpful for us. We basically outline the kind of person we are not looking for. The point here is to screen out people who on paper might look like a great fit, but because of some specific reason, this particular role wouldn't be good for them. 
+
+This isn't where you put 'we are not looking for lazy people who are bad at their jobs', more 'This role isn't a great fit if you are looking to eventually manage a large team' (or whatever). 
+
+You can also do a version of this list under a 'what you'll be doing' type section - it's super helpful to make explicit what the candidate won't be doing. Again, this helps candidates to screen themselves out. As I said before, you are not that great a negotiator - if you hire someone and force them to do something they don't want to do, they will quite in 6 months anyway and you'll have to repeat this all again. 
+
+**If you are open source, link to your repo**
+
+Ok this is more niche, but if your product is open source then linking to good examples of recent PRs/Issues is a great way to give candidates an insight into exactly the kind of thing they will be working on. Instead of promising them 'hey you'll work on something really cool, trust me bro' you can actually point them to real code. Neat!
```

Comment:
```suggestion
This is more niche, but if your product is open source then linking to good examples of recent PRs/Issues is a great way to give candidates an insight into exactly the kind of thing they will be working on. Instead of promising them, 'Hey you'll work on something really cool, trust me' you can actually point them to real code. Neat!
```

Reply:
Left in 'bro' ([intentional joke](https://www.urbandictionary.com/define.php?term=trust%20me%20bro))